docs/sources/developers/plugins/plugin.schema.json

```diff
@@ -477,6 +477,10 @@
     "tracing": {
       "type": "boolean",
       "description": "For data source plugins, if the plugin supports tracing. Used for example to link logs (e.g. Loki logs) with tracing plugins."
+    },
+    "hasRequiredConfigFields": {
+      "type": "boolean",
+      "description": "For data source plugins, if the plugin has required fields marked with * in the config. Used to render info about required fields on top of the config page."
     }
```

Comment:
I don't think this is a good place for this, this should be handled in React

Reply:
+1 on this.